composer.json

```diff
@@ -12,7 +12,7 @@
         "symfony/options-resolver": "~2.6 || ~3.0 || ^4.0"
   },
   "type": "magento2-module",
-  "version": "3.1.4",
+  "version": "3.1.3",
```

Comment:
Don't put the version in composer.json, composer can read git tags so this is just a source for mistakes for us.